update default logo

kielni opened the pull request:
fixes https://github.com/RagtagOpen/nomad/issues/652

add Jane Doe logo as logo-default.png
update default CSS to use logo-default.png  (override via custom CSS for other uses)

jillh510 commented:
The logo on nomad.ragtag.org should remain the Ragtag logo; this Jane Doe logo should be used on the nomad-demo.ragtag.org site.

kielni commented:
Sorry, misunderstood. Added this logo as `img/logo-demo.png` and added custom CSS that uses it.

After pushing to the demo site, set `BRANDING_CSS_URL` in environment to `/static/css/demo.css`

flynnwastaken commented:
As far as I can tell this didn't go through, see: https://nomad-demo.ragtag.org
